docs/api/framework.rst

```diff
@@ -9,3 +9,4 @@ Framework-specific API
 .. toctree::
 
     pytorch
+    jax
```

Comment:
The main README should also mention JAX.
Grep for pytorch in that document and add JAX at those places and add a JAX example too as there is a PyTorch example.

Reply:
I think README.md need more refactor to show all FWs supported by TE. Diretort adding JAX to all Pytorch appearance might mess up reading. We can submit a split PR to refactor README.md.